src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -190,6 +224,19 @@ private void updateCaseInCcd(
                 existingCase.getId(),
                 exceptionRecord.id
             );
+
+            return Either.right(true);
+        } catch (FeignException.UnprocessableEntity exception) {
```

Comment:
do we have a test case for this ?

Reply:
It was caught for logging purposes, then decided to leave it here, I will add test case for this.